include/cpp_bindgen/fortran_array_view.hpp

```diff
@@ -160,18 +169,21 @@ namespace cpp_bindgen {
         enable_if_t<std::is_lvalue_reference<T>::value && std::is_array<remove_reference_t<T>>::value &&
                     std::is_arithmetic<remove_all_extents_t<remove_reference_t<T>>>::value>> : std::true_type {};
 
+    // simplify this when support for CUDA 9.2 is gone
```

Comment:
It can also be simplified once legacy support is removed?

Reply:
yes